src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -150,8 +151,48 @@ public void run() {
         driver.sendStatusUpdate(status);
     }
 
+    private boolean changeOwnership(String directory, String user, String group) throws IOException, InterruptedException {
+        ProcessBuilder changeOwnershipProceesBuilder = new ProcessBuilder("sudo", "/bin/chown",
+                "-R", user + ":" + group, directory);
+        changeOwnershipProceesBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT);
+        changeOwnershipProceesBuilder.redirectError(ProcessBuilder.Redirect.INHERIT);
+        Process changeOwnershipProcess = changeOwnershipProceesBuilder.start();
+        return changeOwnershipProcess.waitFor() == 0;
+    }
+
+    //Not particularly pleased I have to do this ...
+    private boolean setSuidBit(String directory) throws IOException, InterruptedException {
+        ProcessBuilder setPermsProcessBuilder = new ProcessBuilder("sudo", "/bin/chmod", "3050",
+                directory + "/bin/container-executor");
+        setPermsProcessBuilder.redirectOutput(ProcessBuilder.Redirect.INHERIT);
+        setPermsProcessBuilder.redirectError(ProcessBuilder.Redirect.INHERIT);
+        Process setPermsProcess = setPermsProcessBuilder.start();
+        return setPermsProcess.waitFor() == 0;
+    }
+
+    //@Todo: Discuss Java 7 as it makes changing POSIX file perms way easier, is it worth the dependency?
```

Comment:
I think it's safe to assume Java 7 at this point. We've been doing so in other Java/Scala frameworks, and we're already setting the dependency as such in `build.gradle`.